includes/rest/class-followers-controller.php

```diff
@@ -0,0 +1,290 @@
+							'type'        => 'integer',
+							'default'     => 1,
+							'minimum'     => 1,
+						),
+						'per_page' => array(
+							'description' => 'Maximum number of items to be returned in result set.',
+							'type'        => 'integer',
+							'default'     => 20,
+							'minimum'     => 1,
+						),
+						'order'    => array(
+							'description' => 'Order sort attribute ascending or descending.',
+							'type'        => 'string',
+							'default'     => 'desc',
+							'enum'        => array( 'asc', 'desc' ),
+						),
+						'context'  => array(
+							'description' => 'The context in which the request is made.',
+							'type'        => 'string',
+							'default'     => 'simple',
+							'enum'        => array( 'simple', 'full' ),
+						),
+					),
+				),
+				'schema' => array( $this, 'get_item_schema' ),
+			)
+		);
+	}
+
+	/**
+	 * Retrieves followers list.
+	 *
+	 * @param \WP_REST_Request $request Full details about the request.
+	 * @return \WP_REST_Response|\WP_Error Response object on success, or WP_Error object on failure.
+	 */
+	public function get_items( $request ) {
+		$user_id = $request->get_param( 'user_id' );
+		$user    = Actors::get_by_various( $user_id );
+
+		if ( \is_wp_error( $user ) ) {
+			return $user;
+		}
+
+		/**
+		 * Action triggered prior to the ActivityPub profile being created and sent to the client.
+		 */
+		\do_action( 'activitypub_rest_followers_pre' );
+
+		$order    = $request->get_param( 'order' );
+		$per_page = $request->get_param( 'per_page' );
+		$page     = $request->get_param( 'page' );
+		$context  = $request->get_param( 'context' );
+
+		$data = Follower_Collection::get_followers_with_count( $user_id, $per_page, $page, array( 'order' => \ucwords( $order ) ) );
+
+		$response = array(
+			'@context'     => \Activitypub\get_context(),
+			'id'           => get_rest_url_by_path( \sprintf( 'actors/%d/followers', $user->get__id() ) ),
+			'generator'    => 'https://wordpress.org/?v=' . get_masked_wp_version(),
+			'actor'        => $user->get_id(),
+			'type'         => 'OrderedCollectionPage',
+			'totalItems'   => $data['total'],
+			'partOf'       => get_rest_url_by_path( \sprintf( 'actors/%d/followers', $user->get__id() ) ),
+			'orderedItems' => array_map(
+				function ( $item ) use ( $context ) {
+					if ( 'full' === $context ) {
+						return $item->to_array( false );
+					}
+					return $item->get_id();
+				},
+				$data['followers']
+			),
+		);
+
+		if ( $page && ( ( \ceil( $response['totalItems'] / $per_page ) ) > $page ) ) {
+			$response['next'] = \add_query_arg( 'page', $page + 1, $response['partOf'] );
+		}
+
+		if ( $page && ( $page > 1 ) ) {
+			$response['prev'] = \add_query_arg( 'page', $page - 1, $response['partOf'] );
+		}
+
+		$response['first'] = \add_query_arg( 'page', 1, $response['partOf'] );
+		$response['last']  = \add_query_arg( 'page', \ceil( $response['totalItems'] / $per_page ), $response['partOf'] );
```

Comment:
by default (no followers) it shows `0` and loading that page throws an error. either this should link to min `1` or only show up if $data has more than one page.